mpa/cls/stage.py

```diff
@@ -299,6 +301,26 @@ def configure_hyperparams(cfg, training, **kwargs):
             if lr is not None:
                 cfg.optimizer.lr = lr
 
+    def _put_model_on_gpu(self, model, cfg):
```

Comment:
Is this function only used for classification? I wonder if it can be used for other tasks as well.

Reply:
No, it can't. In other tasks, similar things are done in each framework (mmdet, mmseg).